src/applications/common/schemaform/fields/AutosuggestField.jsx

```diff
@@ -14,6 +14,13 @@ function getInput(input, uiSchema, schema) {
 
   if (typeof input !== 'object' && input) {
     const uiOptions = uiSchema['ui:options'];
+    // When using this field in an array item, editing the item will throw an error
+    //  if there uiOptions.label is undefined (as when we queryForResults), so we
+    //  have to have this safety valve
+    if (!uiOptions.labels) {
+      return input;
```

Comment:
@jbalboni I don't think this will break anything, but I'm not sure how to be _certain_ it won't.

Reply:
Looks good to me.